hardware/raspberrypi/gpio/gpio_pads_control2.md

```diff
@@ -0,0 +1,92 @@
+# GPIO pads control
+
+Extracted from BCM2835 full data sheet, from Gert van Loo 2-August-2012. Rewritten into this markdown document by Robert Bisewski in 21-October-2018.
+
```

Comment:
We don't have any attribution (except CC stuff at bottom of pages) in our documentation so would prefer this section be removed.